pages/index.tsx

```diff
@@ -256,27 +256,23 @@ export default function Home(props: {
       {/* Featuring Keynotes speakers */}
 
       {speakers.length != 0 && (
-        <section className="flex overflow-x-auto bg-gray-200 min-h-[24rem]">
-          <div className="flex items-center justify-center font-bold p-6 md:text-4xl text-2xl my-4">
+        <section className=" overflow-x-auto min-h-[24rem]">
+          <div className="flex items-start justify-start font-bold p-6 md:text-4xl text-2xl my-4">
             Featuring Keynote Speakers
           </div>
-          <div className="flex flex-col justify-center py-6 md:px-6">
-            {/* Row 1 */}
-            <div className="flex">
-              {speakers.map(
-                ({ name, description, fileName }, idx) =>
-                  idx < speakers.length / 2 && (
-                    <KeynoteSpeaker
-                      key={idx}
-                      name={name}
-                      description={description}
-                      cardColor={colorSchemes[idx % 3]}
-                      imageLink={fileName}
-                    />
-                  ),
-              )}
-            </div>
-            {/* row 2 */}
+
+          <div className="flex md:flex-row flex-col items-center justify-center font-bold p-6 md:text-4xl text-2xl my-4 md:gap-24 gap-12">
+            {speakers.map(({ name, description, fileName }, idx) => (
+              <KeynoteSpeaker
+                key={idx}
+                name={name}
+                description={description}
+                cardColor={colorSchemes[idx % 3]}
+                imageLink={fileName}
+              />
+            ))}
+          </div>
+          {/* row 2
```

Comment:
this is prob fine but will almost 100% create merge conflicts with #129

Reply:
so you want me to remove it?